onnxscript/version_converter/version_converter.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+from __future__ import annotations
+
+from onnxscript import ir
+
+### Adapters
+
+# Compatibility Adapter
+def adapter_compatible(op: ir.Node, target_opset):
```

Comment:
nit: I recommend `node` instead of `op`. We frequently use `op` for building a node via the `op.MatMul(x, y)` syntax.

Reply:
Furthermore, the general interface for a version-converter would likely need an op/node builder as an input, and that would be best called `op`. We would need that when creating new nodes as part of the version-conversion (eg., even in the example below to create a `Constant` node from an attribute value.